playbooks/roles/deploy-osh/tasks/helm-run.yml

```diff
@@ -1,63 +1,51 @@
 ---
 # Inspired by https://github.com/openstack/openstack-helm-infra/blob/master/roles/build-helm-packages/tasks/setup-helm-serve.yaml
-- block:
-    - name: checking if local helm server is running
-      wait_for:
-        delay: 5
-        timeout: 10
-        host: 127.0.0.1
-        port: 8879
-        state: started
-        msg: "Failure to connect on helm server -- Attempting automatic recovery"
-      register: _server_running
+- name: Check if systemd unit file is present
+  stat:
+    path: /etc/systemd/system/helm-serve.service
+  register: helm_serve_service
 
-    - name: Check port 8879 is running a helm repository
-      uri:
-        url: http://127.0.0.1:8879
-        return_content: yes
-      register: this
-      failed_when: "'Helm Repository' not in this.content"
+- name: Copy systemd unit into place for helm server if it is not present
+  when: not helm_serve_service.stat.exists
```

Comment:
We probably don't need to check if it exists beforehand. This command should be safe to run idempotent and will ensure we get the latest template.

Reply:
I'm not sure. I think the original point was, let's have systemd unit file localy as a backup, but in future we want it to have packaged.
So this way it only uses the local copy for the cases it wasn't packaged (yet)